bootstrap-app/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:7
+
+# Install Java and clean up.
+RUN curl -LO --cookie 'oraclelicense=accept-securebackup-cookie' 'http://download.oracle.com/otn-pub/java/jdk/8u45-b14/jdk-8u45-linux-x64.rpm' \
+ && rpm -Uvh jdk-8u45-linux-x64.rpm \
+ && rm jdk-8u45-linux-x64.rpm
+ENV JAVA_HOME /usr/java/jdk1.8.0_45/jre
+
+COPY . target/cmr-bootstrap-app-0.1.0-SNAPSHOT-standalone.jar
```

Comment:
This copy command should be copying the jar to /app/ like the other Dockerfiles

Reply:
Good catch - fixed